project/ReleaseUtils.scala

```diff
@@ -146,7 +143,7 @@ object ReleaseUtils {
        |      mkdir "channel"
        |
        |      mv "${artifacts.bloopCoursier.name}", "channel/bloop.json"
-       |      system "coursier", "install", "--install-dir", "bin", "--default-channels=false", "--channel", "channel", "bloop", "-J-Divy.home=$ivyHome"
```

Comment:
This would add ivy home for gihub runner :O

Reply:
How strange 🤔